SCClassLibrary/Common/Quarks/Quark.sc

```diff
@@ -78,6 +78,13 @@ Quark {
 		});
 		^isCompatible
 	}
+	runHook { |hook|
+		hook = hook.asSymbol;
+		if(this.data[hook].notNil, {
+			"Run % hook".format(hook).postln;
+			this.data[hook].();
```

Comment:
Consider wrapping the hook callback in a `try` with a more specific error message? I don't think we can totally protect or clean-up in case of an error while calling a hook, but it would be good to report errors as something more than a random sclang exception (these are hard enough interpret for people....). I'm imagining something like this would be enough:
```
try {
  this.data[hook].value();
} {
   |e|
   "Encountered error while running hook '%' for Quark('%'). Installation or update for this Quark may not have completed correctly.".warn;
   e.throw(); // re-throw
}